SpatialGDK/Source/SpatialGDK/Public/SpatialView/ReceivedOpEventHandler.h

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/Connection/SpatialEventTracer.h"
+#include "OpList/OpList.h"
+
+namespace SpatialGDK
+{
+class FReceivedOpEventHandler
+{
+public:
+	explicit FReceivedOpEventHandler(TSharedPtr<SpatialEventTracer> EventTracer = nullptr);
+	void ProcessOpLists(const OpList& Ops);
+
+private:
+	void HandleAddEntity();
```

Comment:
Do we need these functions?

Reply:
Ah they were meant to be deleted. Will do that.